packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -422,9 +414,10 @@ class OverflowMenu extends Component {
   _handlePlace = menuBody => {
     if (menuBody) {
       this._menuBody = menuBody;
-      (
-        menuBody.querySelector('[data-floating-menu-primary-focus]') || menuBody
-      ).focus();
+      const primaryFocus =
```

Comment:
What do you think the best name for this would be? Would it be something like `focusTarget`? Something else? curious what you feel like this is for and what name could communicate that clearly.

Reply:
I think `primaryFocus` works! In OverflowMenuItem, it seems like they use `primaryFocus` to mean a similar thing? 
```    const primaryFocusProp = primaryFocus
      ? { 'data-floating-menu-primary-focus': true }
      : {};
```